spec/factories/casa_orgs.rb

```diff
@@ -12,5 +12,15 @@
     trait :with_logo do
       logo { Rack::Test::UploadedFile.new(Rails.root.join("spec", "fixtures", "org_logo.jpeg")) }
     end
+
+    trait :with_placement_types do
+      transient { placement_types { ["Reunification", "Adoption", "Foster Care", "Kinship"] } }
```

Comment:
```suggestion
      transient { placement_names { ["Reunification", "Adoption", "Foster Care", "Kinship"] } }
```
Confusing, these aren't placement types, they are just type names.